acceptancetests/jujupy/client.py

```diff
@@ -103,6 +103,7 @@
 LXC_MACHINE = 'lxc'
 LXD_MACHINE = 'lxd'
 
+_DEFAULT_POLL_TIMEOUT = 2
```

Comment:
Can we change this to 5 sec instead? It won't make a big difference in the grand scheme of things but seems a rounder number...